_product/ce/developer-board-4-version-3/README.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Developer Board 4 Version 3
+shortname: developer-board-4-version-3
+permalink: /product/developer-board-4-version-3/
+product_specification: ce
+description: >-
+  The Developer Board 4 Version3 is a 96Boards compliant community board based
+  on Qualcomm® Snapdragon Quad-core ARM Cortex-A53 64-bit application processors
+  with 512kB L2 cache,1GB or 2GB LPDDR3 533MHz,8GB or 16GB eMMC 5.0,support WiFi
+  & LAN, USB, SD/TF card external storage.The mezzaning board defines a 60Pin
+  high-speed connector, a 40pin low-speed connector and a 16pin analog
+  connector.
+product_short_desc: >-
+  The Developer Board 4 Version3 is a 96Boards compliant community board based
+  on Qualcomm® Snapdragon Quad-core ARM Cortex-A53.
+keywords: >-
+  Developer Board, Wi-Fi, Bluetooth connectivity, development, board, low cost,
+  Product, Development, Platform, Qualcomm, high-speed connector,  functional
+  modules
+dateAdded: '2020-01-23 04:13:57'
+product_images: images/96boards-banner-project-2.jpg
+image: /assets/images/content/96Boards-Logo_v-Consumer.png
```

Comment:
wrong image path.
You have added the image to  _product/ce/developer-board-4-version-3/images/96Boards-Logo_v-Consumer.png but you mention /assets/images/content/96Boards-Logo_v-Consumer.png